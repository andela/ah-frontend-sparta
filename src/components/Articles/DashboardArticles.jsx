import React from 'react';
import PropTypes from 'prop-types';
import moment from 'moment';
import DisplayArticles from './DisplayArticles';
import './article.scss';


const DashboardArticles = ({ firstArticles, totalArticles }) => (

  <div>
    <p className="text">
      Showing
      {' '}
      <strong>{firstArticles && firstArticles.length}</strong>
      {' '}
      out of
      {' '}
      <strong>{totalArticles && totalArticles.length}</strong>
    </p>
    {
      totalArticles.length > 0 ? (
        totalArticles.map((article) => {
          const articleDate = moment(article.createdAt.slice(0, 10)).format(
            'LL',
          );
          return (
            <div key={article.id} className="container" style={{ marginTop: '20px' }}>
              <DisplayArticles
                article={article}
                articleDate={articleDate}
              />
              <br />
            </div>

          );
        })
      ) : (
        <div>No Posts where Found !!!!</div>
      )}
  </div>

);
DashboardArticles.defaultProps = {
  totalArticles: [],
};
DashboardArticles.propTypes = {
  totalArticles: PropTypes.arrayOf(PropTypes.object),
};
export default DashboardArticles;
