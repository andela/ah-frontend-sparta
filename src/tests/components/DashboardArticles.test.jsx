import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import DashboardArticles from '../../components/Articles/DashboardArticles';

describe('Dashboard Articles Component', () => {
  let mountApp;
  const props = {
    totalArticles: [

      {
        id: 1,
        author: {
          username: 'rogha',
          firstname: null,
          lastname: null,
          bio: null,
          image: null,
          followers_no: 1,
          following_no: 0,
        },
        average_rating: null,
        title: 'Post one',
        description: 'too long a story',
        slug: 'post-one-first',
        body: 'ahhhhh --edit me',
        createdAt: '2019-04-18T11:55:58.111843Z',
        updatedAt: '2019-04-18T11:55:58.111902Z',
        tags: [],
        likes: 0,
        dislikes: 0,
      },
      {
        id: 2,
        author: {
          username: 'rogha',
          firstname: null,
          lastname: null,
          bio: null,
          image: null,
          followers_no: 1,
          following_no: 0,
        },
        average_rating: null,
        title: 'Post one',
        description: 'too long a story',
        slug: 'post-one-first',
        body: 'ahhhhh --edit me',
        createdAt: '2019-04-18T11:55:58.111843Z',
        updatedAt: '2019-04-18T11:55:58.111902Z',
        tags: [],
        likes: 0,
        dislikes: 0,
      },
      {
        id: 3,
        author: {
          username: 'rogha',
          firstname: null,
          lastname: null,
          bio: null,
          image: null,
          followers_no: 1,
          following_no: 0,
        },
        average_rating: null,
        title: 'Post one',
        description: 'too long a story',
        slug: 'post-one-first',
        body: 'ahhhhh --edit me',
        createdAt: '2019-04-18T11:55:58.111843Z',
        updatedAt: '2019-04-18T11:55:58.111902Z',
        tags: [],
        likes: 0,
        dislikes: 0,
      },
      {
        id: 4,
        author: {
          username: 'rogha',
          firstname: null,
          lastname: null,
          bio: null,
          image: null,
          followers_no: 1,
          following_no: 0,
        },
        average_rating: null,
        title: 'Post one',
        description: 'too long a story',
        slug: 'post-one-first',
        body: 'ahhhhh --edit me',
        createdAt: '2019-04-18T11:55:58.111843Z',
        updatedAt: '2019-04-18T11:55:58.111902Z',
        tags: [],
        likes: 0,
        dislikes: 0,
      },
      {
        id: 5,
        author: {
          username: 'rogha',
          firstname: null,
          lastname: null,
          bio: null,
          image: null,
          followers_no: 1,
          following_no: 0,
        },
        average_rating: null,
        title: 'Post one',
        description: 'too long a story',
        slug: 'post-one-first',
        body: 'ahhhhh --edit me',
        createdAt: '2019-04-18T11:55:58.111843Z',
        updatedAt: '2019-04-18T11:55:58.111902Z',
        tags: [],
        likes: 0,
        dislikes: 0,
      },
      {
        id: 6,
        author: {
          username: 'rogha',
          firstname: null,
          lastname: null,
          bio: null,
          image: null,
          followers_no: 1,
          following_no: 0,
        },
        average_rating: null,
        title: 'Post one',
        description: 'too long a story',
        slug: 'post-one-first',
        body: 'ahhhhh --edit me',
        createdAt: '2019-04-18T11:55:58.111843Z',
        updatedAt: '2019-04-18T11:55:58.111902Z',
        tags: [],
        likes: 0,
        dislikes: 0,
      },
      {
        id: 7,
        author: {
          username: 'rogha',
          firstname: null,
          lastname: null,
          bio: null,
          image: null,
          followers_no: 1,
          following_no: 0,
        },
        average_rating: null,
        title: 'Post one',
        description: 'too long a story',
        slug: 'post-one-first',
        body: 'ahhhhh --edit me',
        createdAt: '2019-04-18T11:55:58.111843Z',
        updatedAt: '2019-04-18T11:55:58.111902Z',
        tags: [],
        likes: 0,
        dislikes: 0,
      },
    ],
  };
  beforeEach(() => {
    mountApp = renderer.create(
      <Router>
        <DashboardArticles {...props} />
      </Router>,
    );
  });

  it('Should render without crashing', () => {
    expect(mountApp).toMatchSnapshot();
  });
});
