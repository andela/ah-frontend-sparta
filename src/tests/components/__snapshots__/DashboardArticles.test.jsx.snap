// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Articles Component Should render without crashing 1`] = `
<div>
  <p
    className="text"
  >
    Showing
     
    <strong />
     
    out of
     
    <strong>
      7
    </strong>
  </p>
  <div
    className="container"
    style={
      Object {
        "marginTop": "20px",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="row"
        >
          <div
            className="card col-md-12"
            style={
              Object {
                "boxShadow": "5px 10px 18px #888888",
              }
            }
          >
            <h5
              className="card-header"
            >
              Post one
            </h5>
            <div
              className="card-body"
            >
              <h5
                className="card-title"
              >
                Written on
                 
                April 18, 2019
              </h5>
              <p
                className="card-text"
              >
                too long a story
              </p>
              <p
                className="read_time"
              />
              <p
                className="author_name"
              >
                Created By
                 
                <b>
                  rogha
                </b>
              </p>
              
              <br />
              <div
                className="card-footer custom-footer"
              >
                <a
                  className="btn btn-primary float-left read-more-button"
                  href="/articles/post-one-first"
                  onClick={[Function]}
                >
                  Read More ...
                </a>
                <div
                  className="float-right"
                >
                  <div
                    className="icons"
                  >
                    <div
                      className="btn-group dropleft"
                    >
                      <a
                        aria-expanded="false"
                        aria-haspopup="true"
                        data-toggle="dropdown"
                        href="sharing"
                      >
                        <i
                          className="fas fa-share-alt-square"
                        />
                      </a>
                      <div
                        className="dropdown-menu"
                      >
                        <a
                          className="dropdown-item"
                          href="https://www.facebook.com/sharer/sharer.php?u=undefined/articles/post-one-first"
                        >
                          <i
                            className="fab fa-facebook-square facebook-icon"
                          />
                          Facebook
                        </a>
                        <a
                          className="dropdown-item"
                          href="https://twitter.com/intent/tweet?url=undefined/articles/post-one-first&text=too long a story"
                        >
                          <i
                            className="fab fa-twitter twitter-icon"
                          />
                          Twitter
                        </a>
                        <a
                          className="dropdown-item"
                          href="mailto:?Subject=Post%20one&body=I found this article to be worth reading, Post one undefined/articles/post-one-first"
                        >
                          <i
                            className="fas fa-envelope-square envelop-icon"
                          />
                          Mail
                        </a>
                      </div>
                    </div>
                       
                    <a
                      href="comment"
                    >
                      <i
                        className="far fa-comment"
                      />
                    </a>
                       
                    <a
                      href="like"
                    >
                      <i
                        className="far fa-thumbs-up pull float-right"
                      />
                    </a>
                       
                    <a
                      href="dislike"
                    >
                      <i
                        className="far fa-thumbs-down"
                      />
                    </a>
                       
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <br />
  </div>
  <div
    className="container"
    style={
      Object {
        "marginTop": "20px",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="row"
        >
          <div
            className="card col-md-12"
            style={
              Object {
                "boxShadow": "5px 10px 18px #888888",
              }
            }
          >
            <h5
              className="card-header"
            >
              Post one
            </h5>
            <div
              className="card-body"
            >
              <h5
                className="card-title"
              >
                Written on
                 
                April 18, 2019
              </h5>
              <p
                className="card-text"
              >
                too long a story
              </p>
              <p
                className="read_time"
              />
              <p
                className="author_name"
              >
                Created By
                 
                <b>
                  rogha
                </b>
              </p>
              
              <br />
              <div
                className="card-footer custom-footer"
              >
                <a
                  className="btn btn-primary float-left read-more-button"
                  href="/articles/post-one-first"
                  onClick={[Function]}
                >
                  Read More ...
                </a>
                <div
                  className="float-right"
                >
                  <div
                    className="icons"
                  >
                    <div
                      className="btn-group dropleft"
                    >
                      <a
                        aria-expanded="false"
                        aria-haspopup="true"
                        data-toggle="dropdown"
                        href="sharing"
                      >
                        <i
                          className="fas fa-share-alt-square"
                        />
                      </a>
                      <div
                        className="dropdown-menu"
                      >
                        <a
                          className="dropdown-item"
                          href="https://www.facebook.com/sharer/sharer.php?u=undefined/articles/post-one-first"
                        >
                          <i
                            className="fab fa-facebook-square facebook-icon"
                          />
                          Facebook
                        </a>
                        <a
                          className="dropdown-item"
                          href="https://twitter.com/intent/tweet?url=undefined/articles/post-one-first&text=too long a story"
                        >
                          <i
                            className="fab fa-twitter twitter-icon"
                          />
                          Twitter
                        </a>
                        <a
                          className="dropdown-item"
                          href="mailto:?Subject=Post%20one&body=I found this article to be worth reading, Post one undefined/articles/post-one-first"
                        >
                          <i
                            className="fas fa-envelope-square envelop-icon"
                          />
                          Mail
                        </a>
                      </div>
                    </div>
                       
                    <a
                      href="comment"
                    >
                      <i
                        className="far fa-comment"
                      />
                    </a>
                       
                    <a
                      href="like"
                    >
                      <i
                        className="far fa-thumbs-up pull float-right"
                      />
                    </a>
                       
                    <a
                      href="dislike"
                    >
                      <i
                        className="far fa-thumbs-down"
                      />
                    </a>
                       
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <br />
  </div>
  <div
    className="container"
    style={
      Object {
        "marginTop": "20px",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="row"
        >
          <div
            className="card col-md-12"
            style={
              Object {
                "boxShadow": "5px 10px 18px #888888",
              }
            }
          >
            <h5
              className="card-header"
            >
              Post one
            </h5>
            <div
              className="card-body"
            >
              <h5
                className="card-title"
              >
                Written on
                 
                April 18, 2019
              </h5>
              <p
                className="card-text"
              >
                too long a story
              </p>
              <p
                className="read_time"
              />
              <p
                className="author_name"
              >
                Created By
                 
                <b>
                  rogha
                </b>
              </p>
              
              <br />
              <div
                className="card-footer custom-footer"
              >
                <a
                  className="btn btn-primary float-left read-more-button"
                  href="/articles/post-one-first"
                  onClick={[Function]}
                >
                  Read More ...
                </a>
                <div
                  className="float-right"
                >
                  <div
                    className="icons"
                  >
                    <div
                      className="btn-group dropleft"
                    >
                      <a
                        aria-expanded="false"
                        aria-haspopup="true"
                        data-toggle="dropdown"
                        href="sharing"
                      >
                        <i
                          className="fas fa-share-alt-square"
                        />
                      </a>
                      <div
                        className="dropdown-menu"
                      >
                        <a
                          className="dropdown-item"
                          href="https://www.facebook.com/sharer/sharer.php?u=undefined/articles/post-one-first"
                        >
                          <i
                            className="fab fa-facebook-square facebook-icon"
                          />
                          Facebook
                        </a>
                        <a
                          className="dropdown-item"
                          href="https://twitter.com/intent/tweet?url=undefined/articles/post-one-first&text=too long a story"
                        >
                          <i
                            className="fab fa-twitter twitter-icon"
                          />
                          Twitter
                        </a>
                        <a
                          className="dropdown-item"
                          href="mailto:?Subject=Post%20one&body=I found this article to be worth reading, Post one undefined/articles/post-one-first"
                        >
                          <i
                            className="fas fa-envelope-square envelop-icon"
                          />
                          Mail
                        </a>
                      </div>
                    </div>
                       
                    <a
                      href="comment"
                    >
                      <i
                        className="far fa-comment"
                      />
                    </a>
                       
                    <a
                      href="like"
                    >
                      <i
                        className="far fa-thumbs-up pull float-right"
                      />
                    </a>
                       
                    <a
                      href="dislike"
                    >
                      <i
                        className="far fa-thumbs-down"
                      />
                    </a>
                       
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <br />
  </div>
  <div
    className="container"
    style={
      Object {
        "marginTop": "20px",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="row"
        >
          <div
            className="card col-md-12"
            style={
              Object {
                "boxShadow": "5px 10px 18px #888888",
              }
            }
          >
            <h5
              className="card-header"
            >
              Post one
            </h5>
            <div
              className="card-body"
            >
              <h5
                className="card-title"
              >
                Written on
                 
                April 18, 2019
              </h5>
              <p
                className="card-text"
              >
                too long a story
              </p>
              <p
                className="read_time"
              />
              <p
                className="author_name"
              >
                Created By
                 
                <b>
                  rogha
                </b>
              </p>
              
              <br />
              <div
                className="card-footer custom-footer"
              >
                <a
                  className="btn btn-primary float-left read-more-button"
                  href="/articles/post-one-first"
                  onClick={[Function]}
                >
                  Read More ...
                </a>
                <div
                  className="float-right"
                >
                  <div
                    className="icons"
                  >
                    <div
                      className="btn-group dropleft"
                    >
                      <a
                        aria-expanded="false"
                        aria-haspopup="true"
                        data-toggle="dropdown"
                        href="sharing"
                      >
                        <i
                          className="fas fa-share-alt-square"
                        />
                      </a>
                      <div
                        className="dropdown-menu"
                      >
                        <a
                          className="dropdown-item"
                          href="https://www.facebook.com/sharer/sharer.php?u=undefined/articles/post-one-first"
                        >
                          <i
                            className="fab fa-facebook-square facebook-icon"
                          />
                          Facebook
                        </a>
                        <a
                          className="dropdown-item"
                          href="https://twitter.com/intent/tweet?url=undefined/articles/post-one-first&text=too long a story"
                        >
                          <i
                            className="fab fa-twitter twitter-icon"
                          />
                          Twitter
                        </a>
                        <a
                          className="dropdown-item"
                          href="mailto:?Subject=Post%20one&body=I found this article to be worth reading, Post one undefined/articles/post-one-first"
                        >
                          <i
                            className="fas fa-envelope-square envelop-icon"
                          />
                          Mail
                        </a>
                      </div>
                    </div>
                       
                    <a
                      href="comment"
                    >
                      <i
                        className="far fa-comment"
                      />
                    </a>
                       
                    <a
                      href="like"
                    >
                      <i
                        className="far fa-thumbs-up pull float-right"
                      />
                    </a>
                       
                    <a
                      href="dislike"
                    >
                      <i
                        className="far fa-thumbs-down"
                      />
                    </a>
                       
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <br />
  </div>
  <div
    className="container"
    style={
      Object {
        "marginTop": "20px",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="row"
        >
          <div
            className="card col-md-12"
            style={
              Object {
                "boxShadow": "5px 10px 18px #888888",
              }
            }
          >
            <h5
              className="card-header"
            >
              Post one
            </h5>
            <div
              className="card-body"
            >
              <h5
                className="card-title"
              >
                Written on
                 
                April 18, 2019
              </h5>
              <p
                className="card-text"
              >
                too long a story
              </p>
              <p
                className="read_time"
              />
              <p
                className="author_name"
              >
                Created By
                 
                <b>
                  rogha
                </b>
              </p>
              
              <br />
              <div
                className="card-footer custom-footer"
              >
                <a
                  className="btn btn-primary float-left read-more-button"
                  href="/articles/post-one-first"
                  onClick={[Function]}
                >
                  Read More ...
                </a>
                <div
                  className="float-right"
                >
                  <div
                    className="icons"
                  >
                    <div
                      className="btn-group dropleft"
                    >
                      <a
                        aria-expanded="false"
                        aria-haspopup="true"
                        data-toggle="dropdown"
                        href="sharing"
                      >
                        <i
                          className="fas fa-share-alt-square"
                        />
                      </a>
                      <div
                        className="dropdown-menu"
                      >
                        <a
                          className="dropdown-item"
                          href="https://www.facebook.com/sharer/sharer.php?u=undefined/articles/post-one-first"
                        >
                          <i
                            className="fab fa-facebook-square facebook-icon"
                          />
                          Facebook
                        </a>
                        <a
                          className="dropdown-item"
                          href="https://twitter.com/intent/tweet?url=undefined/articles/post-one-first&text=too long a story"
                        >
                          <i
                            className="fab fa-twitter twitter-icon"
                          />
                          Twitter
                        </a>
                        <a
                          className="dropdown-item"
                          href="mailto:?Subject=Post%20one&body=I found this article to be worth reading, Post one undefined/articles/post-one-first"
                        >
                          <i
                            className="fas fa-envelope-square envelop-icon"
                          />
                          Mail
                        </a>
                      </div>
                    </div>
                       
                    <a
                      href="comment"
                    >
                      <i
                        className="far fa-comment"
                      />
                    </a>
                       
                    <a
                      href="like"
                    >
                      <i
                        className="far fa-thumbs-up pull float-right"
                      />
                    </a>
                       
                    <a
                      href="dislike"
                    >
                      <i
                        className="far fa-thumbs-down"
                      />
                    </a>
                       
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <br />
  </div>
  <div
    className="container"
    style={
      Object {
        "marginTop": "20px",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="row"
        >
          <div
            className="card col-md-12"
            style={
              Object {
                "boxShadow": "5px 10px 18px #888888",
              }
            }
          >
            <h5
              className="card-header"
            >
              Post one
            </h5>
            <div
              className="card-body"
            >
              <h5
                className="card-title"
              >
                Written on
                 
                April 18, 2019
              </h5>
              <p
                className="card-text"
              >
                too long a story
              </p>
              <p
                className="read_time"
              />
              <p
                className="author_name"
              >
                Created By
                 
                <b>
                  rogha
                </b>
              </p>
              
              <br />
              <div
                className="card-footer custom-footer"
              >
                <a
                  className="btn btn-primary float-left read-more-button"
                  href="/articles/post-one-first"
                  onClick={[Function]}
                >
                  Read More ...
                </a>
                <div
                  className="float-right"
                >
                  <div
                    className="icons"
                  >
                    <div
                      className="btn-group dropleft"
                    >
                      <a
                        aria-expanded="false"
                        aria-haspopup="true"
                        data-toggle="dropdown"
                        href="sharing"
                      >
                        <i
                          className="fas fa-share-alt-square"
                        />
                      </a>
                      <div
                        className="dropdown-menu"
                      >
                        <a
                          className="dropdown-item"
                          href="https://www.facebook.com/sharer/sharer.php?u=undefined/articles/post-one-first"
                        >
                          <i
                            className="fab fa-facebook-square facebook-icon"
                          />
                          Facebook
                        </a>
                        <a
                          className="dropdown-item"
                          href="https://twitter.com/intent/tweet?url=undefined/articles/post-one-first&text=too long a story"
                        >
                          <i
                            className="fab fa-twitter twitter-icon"
                          />
                          Twitter
                        </a>
                        <a
                          className="dropdown-item"
                          href="mailto:?Subject=Post%20one&body=I found this article to be worth reading, Post one undefined/articles/post-one-first"
                        >
                          <i
                            className="fas fa-envelope-square envelop-icon"
                          />
                          Mail
                        </a>
                      </div>
                    </div>
                       
                    <a
                      href="comment"
                    >
                      <i
                        className="far fa-comment"
                      />
                    </a>
                       
                    <a
                      href="like"
                    >
                      <i
                        className="far fa-thumbs-up pull float-right"
                      />
                    </a>
                       
                    <a
                      href="dislike"
                    >
                      <i
                        className="far fa-thumbs-down"
                      />
                    </a>
                       
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <br />
  </div>
  <div
    className="container"
    style={
      Object {
        "marginTop": "20px",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="row"
        >
          <div
            className="card col-md-12"
            style={
              Object {
                "boxShadow": "5px 10px 18px #888888",
              }
            }
          >
            <h5
              className="card-header"
            >
              Post one
            </h5>
            <div
              className="card-body"
            >
              <h5
                className="card-title"
              >
                Written on
                 
                April 18, 2019
              </h5>
              <p
                className="card-text"
              >
                too long a story
              </p>
              <p
                className="read_time"
              />
              <p
                className="author_name"
              >
                Created By
                 
                <b>
                  rogha
                </b>
              </p>
              
              <br />
              <div
                className="card-footer custom-footer"
              >
                <a
                  className="btn btn-primary float-left read-more-button"
                  href="/articles/post-one-first"
                  onClick={[Function]}
                >
                  Read More ...
                </a>
                <div
                  className="float-right"
                >
                  <div
                    className="icons"
                  >
                    <div
                      className="btn-group dropleft"
                    >
                      <a
                        aria-expanded="false"
                        aria-haspopup="true"
                        data-toggle="dropdown"
                        href="sharing"
                      >
                        <i
                          className="fas fa-share-alt-square"
                        />
                      </a>
                      <div
                        className="dropdown-menu"
                      >
                        <a
                          className="dropdown-item"
                          href="https://www.facebook.com/sharer/sharer.php?u=undefined/articles/post-one-first"
                        >
                          <i
                            className="fab fa-facebook-square facebook-icon"
                          />
                          Facebook
                        </a>
                        <a
                          className="dropdown-item"
                          href="https://twitter.com/intent/tweet?url=undefined/articles/post-one-first&text=too long a story"
                        >
                          <i
                            className="fab fa-twitter twitter-icon"
                          />
                          Twitter
                        </a>
                        <a
                          className="dropdown-item"
                          href="mailto:?Subject=Post%20one&body=I found this article to be worth reading, Post one undefined/articles/post-one-first"
                        >
                          <i
                            className="fas fa-envelope-square envelop-icon"
                          />
                          Mail
                        </a>
                      </div>
                    </div>
                       
                    <a
                      href="comment"
                    >
                      <i
                        className="far fa-comment"
                      />
                    </a>
                       
                    <a
                      href="like"
                    >
                      <i
                        className="far fa-thumbs-up pull float-right"
                      />
                    </a>
                       
                    <a
                      href="dislike"
                    >
                      <i
                        className="far fa-thumbs-down"
                      />
                    </a>
                       
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <br />
  </div>
</div>
`;
